FifthLesson/VectorProgram/VectorProgram/ProgramRun.cs

```diff
@@ -13,87 +9,132 @@ class ProgramRun
+            ShowVectorMultiplication(vectorFirst, vectorSecond);
+
+            ShowAngleBetweenVectors(vectorFirst, vectorSecond);
+
+            ReadKey();
+        }
 
-            Console.ReadKey();
+        private void ShowAngleBetweenVectors(Vector vectorFirst, Vector vectorSecond)
+        {
+            var angle = vectorFirst.AngleBetweenVectors(vectorSecond);
+            WriteLine($"The angle between {vectorFirst.ToString()} and {vectorSecond.ToString()} = {angle.ToString()}");
         }
 
-        private static void CompareVectors(Vector vectorFirst, Vector vectorSecond)
+        private void CompareVectors(Vector first, Vector second)
         {
-            Console.WriteLine($"{vectorFirst.ToString()} == {vectorSecond.ToString()} = {(vectorFirst == vectorSecond).ToString()}");
-            Console.WriteLine($"{vectorFirst.ToString()} != {vectorSecond.ToString()} = {(vectorFirst != vectorSecond).ToString()}");
+            WriteLine("Compare vectors:");
+            WriteLine($"{first.ToString()} == {second.ToString()} = {(first == second).ToString()}");
+            WriteLine($"{first.ToString()} != {second.ToString()} = {(first != second).ToString()}");
         }
 
-        private void ShowMultiplicationAndVectors(Vector vectorFirst, Vector vectorSecond)
+        private void ShowMultiplicationAndVectors(Vector first, Vector second)
         {
-            Console.WriteLine("Enter a number to produce a vector by a number:");
-            var numericInput = ReadInputDoubleAndVerificate();
-            Console.WriteLine($"{vectorFirst.ToString()} * {numericInput.ToString()} = {(vectorFirst * numericInput).ToString()}");
-            Console.WriteLine($"{vectorSecond.ToString()} * {numericInput.ToString()} = {(vectorSecond * numericInput).ToString()}");
+            var numericMultiplier = GetUserNumericMultiplier();
+            WriteLine("Multiplication of vectors by number:");
+
+            WriteLine($"{first.ToString()} * {numericMultiplier.ToString()} = {(first * numericMultiplier).ToString()}");
+            WriteLine($"{second.ToString()} * {numericMultiplier.ToString()} = {(second * numericMultiplier).ToString()}\n");
 
-            Console.WriteLine($"{numericInput.ToString()} * {vectorFirst.ToString()} = {(numericInput * vectorFirst).ToString()}");
-            Console.WriteLine($"{numericInput.ToString()} * {vectorSecond.ToString()} = {(numericInput * vectorSecond).ToString()}");
+            WriteLine($"{numericMultiplier.ToString()} * {first.ToString()} = {(numericMultiplier * first).ToString()}");
+            WriteLine($"{numericMultiplier.ToString()} * {second.ToString()} = {(numericMultiplier * second).ToString()}");
         }
 
-        private static void ShowVectorsAndActions(Vector vectorFirst, Vector vectorSecond)
+        private void ShowActionsWithVectors(Vector first, Vector second)
```

Comment:
I would never had guessed what does this method do.
Could you rename it, please?